nixos/modules/programs/gpu-screen-recorder.nix

```diff
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+let
+
+  cfg = config.programs.gpu-screen-recorder;
+  package = cfg.package.override {
+    inherit (config.security) wrapperDir;
+  };
+
+in {
+  options = {
+
+    programs.gpu-screen-recorder = {
+      package = lib.mkPackageOption pkgs "gpu-screen-recorder" {};
+
+      wrapCapabilities = lib.mkOption {
+        type = lib.types.bool;
+        default = false;
```

Comment:
Why not default this to true? Also does this even need to be an option?

Reply:
I wouldn't dare to impose automatically generating these wrappers on anyone who is just loading the program module definition and doesn't actually use gpu-screen-recorder (do note that we don't have an `enable` option here, as it would be unclear whether it just installs the package or actually starts the included background service).

Unless you are asking why I haven't put the "generating wrappers" part into `pkgs/...`, in which case the answer would be "I couldn't figure out how".

Reply:
I'm in favor of setting it enabled as default. It's a big boon of nix modules to 'take care' of these sorts of options. Getting prompted for root when starting a replay is kinda annoying, and when it's resolved by just setting `programs.gpu-screen-recorder.enable` to true, that's really nice. If people just want the binaries, they'd just add the package names to their systemPackages. Maybe you can set a note in the description what it does and any security implication it may have?

Reply:
I think the issue here is that changing this setting enabled-by-default will cause every system in the world to create these wrappers and depend on gpu-screen-recorder, because there is no option like `programs.gpu-screen-recorder.enable`. So this would be an absolute no-go.

Reply:
I guess we could rename this option to `programs.gpu-screen-recorder.enable` if that's more consistent, but you'd still have to set an option either way.

Reply:
Updated the option name to `.enable` (and updated the option description accordingly), if flashrom does this then we can't be too far off.